next.config.js

```diff
@@ -6,3 +6,41 @@ const nextConfig = {
 };
 
 module.exports = nextConfig;
+
+
+// Injected content via Sentry wizard below
```

Comment:
can you comment on where this block of code was copied from?

Reply:
this is auto-generated when install and init sentry